### ISSUE TITLE
Mahjong Tile Designs - Circle Suite

### ISSUE THREAD

**Issue by tjablin** [googlefonts/noto-fonts#2351]
### Title 
  > Mahjong Tile Designs - Circle Suite

### Font 
  > NotoSansSymbols2-Regular.ttf

### Where the font came from, and when
Pre-installed

### Font Version
2.003

### OS name and version
Debian GNU/Linux rodete

### Application name and version
  > If the issue is observed using a specific app.

### Issue

The design of the circle suite in the Mahjong Tile Unicode Block looks weird in Noto Sans Symbols 2.

Currently, the sizes of the circles for the four, five, six, seven, and eight of dots are non-uniform. Typically, these circles are the same size and only one, two, and three have larger circles.

Currently, the six of circles features six equally spaced circles in a rectangular two by three layout. Typically, the bottom four circles on the six of circle are arranged in a tight square and the top two circles are offset by a large gap. Without the gap between the two dot and bottom four circles, I think it is difficult to rapidly distinguish the six and eight of circles, particularly at small font sizes. The six of circles typically looks like the eight of circles with the second row of circles deleted.

Currently, the seven of circles has four circles in a tight rectangle on the bottom and three circles at a thirty degree angle on top. Typically, the bottom four circles should be arranged in a square identical to the six of circles. The seven of circles typically looks like the eight of circles with the top four circles replaced with three circles on a diagonal.

Currently, the eight of circles has two columns of four circles separated by a gap. Typically, these two columns touch.

Currently, the nine of circles has narrow gaps between each row of circles. Typically, the top and bottom row of the nine of circles aligns of the top and bottom row of the eight of circles.

Photo of a Typical Mahjong Sets
![Photo from Wikipedia](https://upload.wikimedia.org/wikipedia/commons/1/12/MJTiles_Fullset.png)

### Character data
🀙🀚🀛🀜🀝🀞🀟🀠🀡

U+1F01A - U1F021

### Screenshot
![Screenshot from 2022-05-11 14-35-38](https://user-images.githubusercontent.com/4314234/167951916-991c5d30-be7c-4dd3-baa8-eaf59a66b228.png)

**Comment by ChiefMikeK** [comment]
These are per the UNICODE®️ standard;  
[🀡 - U+1F021 - decodeunicode.org](https://decodeunicode.org/en/u+1F021 )

**Comment by twardoch** [comment]
The Decodeunicode website is not the Unicode standard but an independent project. 

The official source of the example representations of characters are the Unicode charts:

https://unicode.org/charts/PDF/U1F000.pdf

My view is that the Noto font should be improved to make the characters more in-line with the tradition of the game.

**Comment by simoncozens** [comment]
Yeah, the Unicode reference glyphs are also wrong here. We can do better. (I thought maybe there were regional variations, but it looks like what @tjablin describes is applicable to all Mahjongg styles: https://ja.wikipedia.org/wiki/麻雀牌#デザイン )

Thanks for the clear bug report, we will prioritise it and work on it!

**Comment by twardoch** [comment]
One could say that these symbols are a particular form of digits (numerals) — and it’s not surprising that a disambiguation happened in the tradition. Disambiguation is an indicator of maturity of a sign system.
